darts/tests/dataprocessing/transformers/test_mappers.py

```diff
@@ -32,21 +32,44 @@ def inverse_ts_func(ts, x):
     lin_series = linear_timeseries(start_value=1, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'), end_value=12)  # noqa: E501
     zeroes = constant_timeseries(value=0.0, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'))
     tens = constant_timeseries(value=10.0, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'))
+    twenties = constant_timeseries(value=20.0, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'))
 
     def test_mapper(self):
         transformed = self.plus_ten.transform(self.zeroes)
         self.assertEqual(transformed, self.tens)
 
+        # multiple time series
+        series_array = [self.zeroes, self.tens]
+        transformed = self.plus_ten.transform(series_array)
+        self.assertEqual(transformed, [self.tens, self.twenties])
+
```

Comment:
maybe it's too late on a friday but would that make sense?
```suggestion
        inputs = [
            self.zeroes,
            [self.zeroes, self.tens]
        ]
        
        expected_outputs = [
            self.tens,
            [self.tens, self.twenties]
        ]
   
        for to_transform, expected_output in zip(transformed_inputs, expected_outputs)
            transformed = self.plus_ten.transform(to_transform)
            self.assertEqual(transformed, expected_output)
```

you could also consider having a list of tuples (input, expected) instead of a `zip`, which would be a bit more readable. same comment applies for the other functions in that file.